engine.py

```diff
@@ -279,10 +276,10 @@ def in_check(self):
 		"""
 		if self.light_to_move: # lights turn to move
 			# checks light king under attack in respact to its position
-			return self.square_under_attack(self.light_king_pos[0], self.light_king_pos[1])
+			return self.square_under_attack(self.king_pos["kl"][0], self.king_pos["kl"][1])
```

Comment:
Here you can just use

return self.square_under_attack(self.king_pos["kl"])

since it's already a tuple, there is no need of splitting the elements to create another tuple

Do the same for dark king below

Reply:
owk